src/FilterWPQuery.php

```diff
@@ -65,19 +70,18 @@ public static function getFilterPriority() : int
 		return static::$filterPriority;
 	}
 
-
 	/** @inheritdoc */
 	public static function getPosts() : array
 	{
-		//Create 4 mock posts with different titles
+		// Create 4 mock posts with different titles.
 		$mockPosts = [];
-		for ($i = 0; $i <= 3; $i++) {
-			$post = new \WP_Post((new \stdClass()));
-			$post->post_title = "Mock Post $i";
+		for ($postNum = 0; $postNum <= 3; $postNum++) {
+			$post = new WP_Post((new stdClass()));
```

Comment:
@hellofromtonya Don't we need a `\` to use `WP_Post` and `stdClass` from the global namespace?

Reply:
Notice on line [5](https://github.com/caldera-learn/rest-api-search/pull/1/files/7d02cc9f87f801f210ebc7819823b5126352b310#diff-448b8c6bdebfa00ff2731d641aceb803R5) and 6 that we're _importing_ both of those classes from the global namespace into your namespace via `use`.  

In doing so, we can get rid of the `\`.  

Why?

1. It's more readable.  You are being very clear and declarative by _importing_ what you need.
2. It mitigates a typo when forgetting to include the `\`.